### DISCUSSION TITLE
Checking whether a list contains an item

### DISCUSSION THREAD

**Discussion by samnguyen-outreach** [cue-lang/cue#2025] [Q&A] [answered]
Hello, I'm trying to validate whether a list of structs contains a specific struct key value, but it's not working as I expect:

`test.cue`
```cue
import "list"
xs: [...{id: int}]
ids: [for x in xs {x.id}]
hasTwo: list.Contains(ids, 2)
policy: true & hasTwo
```

`test.json`
```json
{
  "xs": [{"id":  1}, {"id":  2}]
}
```

`cue eval` seems to do what I would expect (`policy: true`), but `cue vet` does not:

```
❯ cue eval test.json test.cue
xs: [{
    id: 1
}, {
    id: 2
}]
ids: [1, 2]
hasTwo: true
policy: true


❯ cue vet test.json test.cue 
policy: conflicting values false and true:
    ./test.cue:5:9
    ./test.cue:6:9
    ./test.cue:6:16
```

**Comment by antong** [comment, marked as answer]
The symptoms look like #1483 . I think this is a bug. It works if you rename the .json to .cue, which enables a different vet mode .
